src/recipes/ios-promo/index.js

```diff
@@ -0,0 +1,284 @@
+  let locale = (extras.locale || userstate.locale || "unknown").toLowerCase();
+  let restdays = config.restdays; // Only run once
+  let locales = (config.locales || []).map((x)=>x.toLowerCase());
+
+
+  let geoOK= extras.geoOK || isAustralia();
+
+  // All versions
+
+  // Bad locale
+  //console.log({
+  //  now: now,
+  //  lastRun: lastRun,
+  //  locale: locale,
+  //  restdays: restdays,
+  //  locales: locales,
+  //  getOK: geoOK
+  //});
+
+  if (!hasAny(locales, [locale, "*"])) {
+    events.message(NAME, "bad-locale", {locale: locale, locales: locales});
+    return false;
+  }
+  // Already ran this
+  if (lastRun !== 0) { // This recipe is only showed once
+    events.message(NAME, "already-run", {lastRun: lastRun});
+    return false;
+  }
+
+  if (!geoOK) {
+    events.message(NAME, "bad-geo", {});
+    return false;
+  }
+
+  // Sample
+  let myRng = extras.randomNumber !== undefined ? extras.randomNumber : Math.random();
+
+  if (myRng <= config.sample) {
+    return true;
+  } else {
+    events.message(NAME, "bad-random-number", {randomNumber: myRng});
+    return false;
+  }
+};
+
+// run / do
+let run = function (state, extras = {}) {
+  // TODO: We may target different messages by OS / Channel
+
+  let now = extras.when || Date.now();
+  let delay = extras.delay || DELAY;
+  let branch = extras.branch || BRANCH.branch;
+
+  eData.data.lastRun = now;
+  eData.store();
+
+  let flowid = extras.flow_id || uuid();
+  let local = {
+    flow_id: flowid,
+    variation_id: branch.name,
+    survey_id: NAME
+  };
+
+  let storeFlow = function (flow_id, flow) {
+    eData.data.flows[flow_id] = flow.data;
+    eData.store();
+  }.bind(null, local.flow_id);
+
+  /* Commented b/c we will use Telemetry instead of phonehome
+     and assume 100% of all users in locale sampled.
+
+  let maybePhonehome = function (flow) {
+    if (!extras.simulate) {
+      let myRng = ;
+      if (Math.random() < PHONEHOMEPCT) {
+        phonehome(flow.data);
+        events.message(flowid, "attempted-phonehome", flow.data);
+      } else {
+        events.message(flowid, "attempted-phonehome", flow.data);
+      }
+    } else {
+      events.message(flowid, "simulated-phonehome", flow.data);
+    }
+  };
+
+  // make and setup flow
+  let flow = new Flow(local);  // create and update
+  flow.began();
+  maybePhonehome(flow);
+  */
+
+  // make and setup flow
+  let flow = new Flow(local);  // create and update
+  flow.began();
+  // maybePhonehome(flow);
+
+  storeFlow(flow);
+  events.message(local.flow_id, "began", flow.data);
+
+  // Add parameters to url
+  let fullUrl = `${branch.url}?source=hb&hbv=${VERSION}` +
+      `&c=${state.updateChannel}&v=${state.fxVersion}&l=${state.locale}` +
+      `&b=${branch.name}`;
+  setTimeout(function() {
+    UITour.showHeartbeat(
+      branch.prompt,
+      branch.thankyou,
+      flowid,
+      fullUrl,
+      null, // learn more text, TODO: should this be "What's this?" or something?
```

Comment:
Speak now if we need an info link.  Personally I think it'd be too confusing next to a "Learn more" button.